src/acceptance-tests-brain/test-scripts/010_insecure_registry_test.rb

```diff
@@ -70,38 +73,72 @@
 
 run 'cf apps'
 
+# With the apps running, now watch (tail) all their logs. (Dumped into helper files)
+# Whenever needed we simply cat the relevant logs into the test output.
+
+Thread.new{
+    run "cf logs uploader > #{tmpdir}/app-uploader.log"
+}.abort_on_exception = true
+
 REGISTRIES.each_pair do |regname, registry_url|
+    Thread.new{
+        run "cf logs #{regname} > #{tmpdir}/app-#{regname}.log"
+    }.abort_on_exception = true
+end
+
+# Wait a bit to have the log tailer thread start properly and settle before doing more.
+sleep 5
+
+REGISTRIES.each_pair do |regname, registry_url|
+    puts "wait for registry to be available ......................................... #{regname}"
     # Wait for the registry to be available
     run_with_retry 60, 1 do
         run "curl -kv #{registry_url}/v2/"
     end
+end
+
```

Comment:
Hmm, found it useful for me to to curl `http://uploader.#{ENV['CF_DOMAIN']}` until that responds with `HTTP 200` too; not sure, though, might just be something local?

Reply:
In the interest of robustness I would say, let us add that.

Reply:
Addressed with commit f2be8c95181e3a6e5e099501d59df6ae6a4d7c48.